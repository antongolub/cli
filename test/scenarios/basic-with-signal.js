'use strict'
require('../..')(main)

function main () {
  console.log('started')
  return new Promise((resolve, reject) => {
    process.kill(process.pid, 'SIGTERM')
    setImmediate(() => {
      console.log('finished')
      resolve()
    })
  })
}
