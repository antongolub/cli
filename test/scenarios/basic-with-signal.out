started
Abnormal exit: SIGTERM
1
